frontend/src/store/controller/selectors.ts

```diff
@@ -196,9 +197,50 @@ export const getAlarmLimitsRequest = createSelector(
   getAlarmLimits,
   (alarmLimits: AlarmLimitsRequestResponse): AlarmLimitsRequest | null => alarmLimits.request,
 );
-export const getAlarmLimitsRequestStandby = createSelector(
+export const getAlarmLimitsRequestDraft = createSelector(
   getAlarmLimits,
-  (alarmLimits: AlarmLimitsRequestResponse): AlarmLimitsRequest | null => alarmLimits.standby,
+  (alarmLimits: AlarmLimitsRequestResponse): AlarmLimitsRequest | null => alarmLimits.draft,
+);
+const alarmLimitsCurrentSelector = (key: string) =>
+  createSelector(getAlarmLimitsCurrent, (alarmLimitsCurrent: AlarmLimits | null): Range | null =>
+    alarmLimitsCurrent === null
+      ? null
+      : ((alarmLimitsCurrent as unknown) as Record<string, Range>)[key],
+  );
+
+export const getAlarmLimitsCurrentSpo2 = alarmLimitsCurrentSelector('spo2');
```

Comment:
Really small nitpick, but let's use `SpO2` rather than `Spo2`, to be consistent with the rest of the code.

Reply:
I think the key in mcu_pb is 'spo2' right?